### PR TITLE
Make event names in slack notifications consistent

### PR DESCRIPTION
**Pull request by arun1595**
##### Description

The event names **START WORK** and **STOP WORK** used in slack notifications don't describe the actual events which are work being started and work being stopped (See screenshot). This PR updates the event names to make it consistent with the rest of the events (WORKER APPROVED, WORKER REJECTED)

image

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

### CODE REVIEW

File: app/dashboard/notifications.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def build_message_for_integration(bounty, event_name):
         pass  # no USD conversion rate
 
     title = bounty.title if bounty.title else bounty.github_url
-    msg = f"*{humanize(event_name.replace('bounty', 'funded_issue'))}*" \
+    msg = f"*{humanize_event_name(event_name))}*" \
```
Comment (arun1595) [inline app/dashboard/notifications.py]: I hate this `event_name` duplication but I couldn't come up with a better name :disappointed:

**Comment by PixelantDesign** [comment]
@mbeacom can you merge this?

File: app/dashboard/utils.py [hunk 2]
```diff
@@ -55,15 +55,29 @@ class NoBountiesException(Exception):
...
+      name (str): The event name
 
     Returns:
-        str: The cleaned representation of the string.
+        str: The humanized representation.
 
     """
-    return text.replace('_', ' ').upper()
+    humanized_event_names = {
+      'new_bounty': 'New funded issue',
```
Comment (mbeacom) [inline app/dashboard/utils.py]: This should be 4 space indentation.

File: app/dashboard/notifications.py (hunk 1 above)
Comment (mbeacom) [inline app/dashboard/notifications.py]: This fails travis because there is an extra `)` here.

File: app/dashboard/utils.py [hunk 3]
```diff
@@ -55,15 +55,29 @@ class NoBountiesException(Exception):
     pass
 
 
-def humanize(text):
-    """Clean a notification string of underscores and capitalize it.
+def humanize_event_name(name):
```
Comment (mbeacom) [inline app/dashboard/utils.py]: Can you write a test for this method in `dashboard/tests/test_dashboard_utils.py` ?

**Comment by mbeacom** [comment]
@PixelantDesign Travis CI / pytest is failing. As soon as the requested changes are turned around, yes.

**Comment by arun1595** [comment]
@mbeacom Sorry about the syntax error, it was totally my bad. I've fixed the indentation issue and also added a test for `humanize_event_name`. Can you PTAL?

**Comment by mbeacom** [comment]
Travis is still failing here - Probably has to do with a previous `master` that was failing... Updating from the latest master.  If builds succeed, will merge.

Thanks

**Comment by arun1595** [comment]
@mbeacom 
> if you could remove the [WOC] from the GH PR title after turning over the review comments, it'll greatly accelerate the followup review

Ah, will keep in mind. Thanks for getting this merged!

**Pull request merged**